packages/react-server/core/util/PageUtil.js

```diff
@@ -174,6 +174,7 @@ function standardizeElements(elements) {
 		.makeArray(elements)
 		.map(e => isRootContainer(e)?flattenForRender(e):e)
 		.reduce((m, e) => m.concat(Array.isArray(e)?e:[e]), [])
+		.map(e => isTheFold(e)?markTheFold():e)
```

Comment:
Yep, exactly.  The normalized elements array contains only root elements and control objects.

This introduces a new control object: `{isTheFold: true}`.